packages/dashboard/src/components/Mnemonic/MnemonicRevealLG.tsx

```diff
@@ -0,0 +1,33 @@
+import { experimentalStyled as styled, Grid, Typography } from '@material-ui/core'
+import { MaskColorVar } from '@masknet/theme'
+
+const WordCard = styled(Typography)(
+    ({ theme }) => `
+    padding: ${theme.spacing(1)};
+    border-radius: 6px;
+    color: ${theme.palette.mode === 'dark' ? MaskColorVar.textPrimary : MaskColorVar.textLink};
+    font-size: 14;
+    background-color: ${MaskColorVar.blue.alpha(0.1)};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`,
+)
+
+export interface MnemonicRevealLGProps {
+    words: string[]
+}
+
+// TODO: merge in MnemonicReveal
+export function MnemonicRevealLG(props: MnemonicRevealLGProps) {
```

Comment:
What does `LG` mean?

Reply:
There have two style MnemonicReveal(LG) in our ui design. The LG is large one size. After discussions with Albert, this MnemonicReveal component was abandoned.  I will merge in one component.